<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Formula</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>816</width>
                <height>76</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="1" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="formulaeLayout"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="PlotWidget" name="graphicsView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
                <horstretch>70</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>200</height>
               </size>
              </property>
             </widget>
             <widget class="GraphicsLayoutWidget" name="errorGraph">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>30</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>70</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>250</height>
            </size>
           </property>
           <property name="title">
            <string>Fitting Control</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="1">
             <widget class="QSlider" name="chunks_horizontalSlider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
              <property name="tracking">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Order</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Overlap</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSlider" name="overlap_horizontalSlider">
              <property name="maximum">
               <number>24</number>
              </property>
              <property name="singleStep">
               <number>6</number>
              </property>
              <property name="tracking">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Chuncks</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="order_label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="overlap_label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="chunks_label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="order_horizontalSlider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>49</number>
              </property>
              <property name="tracking">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="portion_horizontalSlider">
              <property name="minimum">
               <number>50</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="tracking">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>true</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Potion</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="porrtion_label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>30</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Map Control </string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>    X</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Order</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Chuncks</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Overlap</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QPushButton" name="starterrorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="3">
             <widget class="QProgressBar" name="Error_map_progressBar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>    Y</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QRadioButton" name="order_x_radioButton">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="chunks_x_radioButton">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="overlap_x_radioButton">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QRadioButton" name="order_y_radioButton">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="chunks_y_radioButton">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="overlap_y_radioButton">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QLabel" name="label_alert">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionPlot"/>
   <addaction name="separator"/>
   <addaction name="actionSamples"/>
   <addaction name="actionstring"/>
   <addaction name="actionerrormap"/>
  </widget>
  <action name="actionPlot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/signal.ico</normaloff>icons/signal.ico</iconset>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
   <property name="toolTip">
    <string>Plot the signal</string>
   </property>
  </action>
  <action name="actionSamples">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/sample.ico</normaloff>icons/sample.ico</iconset>
   </property>
   <property name="text">
    <string>Samples</string>
   </property>
   <property name="toolTip">
    <string>Show Samples</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>icons/open-folder.png</normaloff>icons/open-folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionstring">
   <property name="text">
    <string>string</string>
   </property>
  </action>
  <action name="actionerrormap">
   <property name="text">
    <string>errormap</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
